README.md

```diff
@@ -1,6 +1,7 @@
 # Mezmo CloudWatch Lambda Function
 
-The Mezmo AWS CloudWatch integration relies on [AWS Lambda](https://aws.amazon.com/documentation/lambda/) to route your [CloudWatch Logs](http://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/WhatIsCloudWatchLogs.html) to Mezmo Pipelines.
+The Mezmo AWS CloudWatch integration relies on [AWS Lambda](https://aws.amazon.com/documentation/lambda/) to route your [CloudWatch Logs](http://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/WhatIsCloudWatchLogs.html) to Mezmo products.
```

Comment:
i'd just remove the word products. 1 product. many features.

Reply:
That's fine, but it does seem like `products` would be the correct term.  Within a product there are features.  And Mezmo is the company, but tbh it's all splitting hairs and we can just remove the word.